release/make_CARLOS_deb.sh

```
@@ -0,0 +1,484 @@
+
+## database switches are needed to provide expected behavior for OSCAR 15
+## enforce UTF-8 encoding so that foreign characters are stored 一種語言永遠不夠 
+## handle 0000-00-00 date errors by rounding to 0001-01-01
+## allow hibernate to alter column names
+## tolerate fields without default values that are not named in the query
+db_switch=\'?characterEncoding=UTF-8\\\&zeroDateTimeBehavior=round\\\&useOldAliasMetadataBehavior=true\\\&jdbcCompliantTruncation=false\'
+
+# and the target of mvn 3 is
+TARGET=carlos-0-SNAPSHOT.war
+
+buildDateTime=date 
```

Shellcheck (0.11.0)

[warning] Use var=$(command) to assign output (or quote to assign string).

(SC2209)